tools/stimulus/functions-R.yaml

```diff
@@ -83,8 +83,10 @@ igraph_adjacency:
     IGNORE: RR
 
 igraph_sparse_adjacency:
+    IGNORE: RC, RR
```

Comment:
Let's keep Stimulus unchanged, we want the autogenerated code to come back after the CRAN release.

Reply:
OK, I only saw this comment after I suggested the opposite in the other thread.